database/db.go

```diff
@@ -217,15 +224,21 @@ func NewDbFromConfig(c *Config, logger *logging.Logger, connectorCallbacks Retry
 		DB:              db,
 		Options:         &c.Options,
 		columnMap:       NewColumnMap(db.Mapper),
-		addr:            addr,
+		addrDescription: addrDescription,
 		logger:          logger,
 		tableSemaphores: make(map[string]*semaphore.Weighted),
 	}, nil
 }
 
-// GetAddr returns the database host:port or Unix socket address.
+// GetAddr returns the Redis connection address in a technical form.
 func (db *DB) GetAddr() string {
-	return db.addr
+	return db.addrDescription
+}
+
+// MarshalLogObject implements zapcore.ObjectMarshaler, adding the database address to each log message.
```

Comment:
I would link to `GetAttr` here so that it's clear which additional information is logged.